yo1k/sandbox/exercises/CSC_algorithms/test_sorting.py

```diff
@@ -0,0 +1,54 @@
+import itertools
+from collections.abc import Sequence, MutableSequence
+from typing import Any
+
+import pytest
+
+import yo1k.sandbox.exercises.CSC_algorithms.sorting as my_sort
```

Comment:
Why are there empty lines here? Is this formatted according to PEP 8?

Reply:
Yes, this is formatted according to [PEP 8](https://peps.python.org/pep-0008/#:~:text=Imports%20should%20be,group%20of%20imports.).

Reply:
Let's discuss. I don't know the PEP, but I am curious to learn the rule that requires this formatting because I fail to derive the rule based on the file - empty lines appear to be placed arbitrarily.

Reply:
Look [here](https://peps.python.org/pep-0008/#:~:text=Imports%20should%20be,group%20of%20imports.).

Reply:
Thanks!, I see.